src/Domain/Fixture/FixtureCreator.php

```diff
@@ -212,6 +212,7 @@ public function create(FixtureOptions $options): void {
     $this->customizeFixture();
     $this->createAndCheckoutBackupTag();
     $this->displayStatus();
+    $this->auditPackages();
```

Comment:
Let's display the `audit` output before the status output, which I think makes the most sense at the end. And Let's change the method name to make it clear we're _displaying_ the audit results, not asserting them, or something--in other words, the result won't fail the build.

```suggestion
    $this->displaySecurityVulnerabilityAdvisories();
    $this->displayStatus();
```